src/Tribe/Aggregator/Event.php

```diff
@@ -188,15 +188,22 @@ public function get_existing_ids( $origin, $values ) {
 		}
 
 		$key = "_{$fields[ $origin ]['target']}";
+		$post_type = Tribe__Events__Main::POSTTYPE;
+		$interval = "('" . implode( "','", $values ) . "')";
```

Comment:
Should we `array_map( 'absint', $values )` for safety?

Reply:
better safe than sorry